dev/NavigationView/NavigationView.cpp

```diff
@@ -1592,6 +1770,161 @@ void NavigationView::UpdateVisualStateForDisplayModeGroup(const winrt::Navigatio
     }
 }
 
+void NavigationView::OnNavigationViewItemTapped(const winrt::IInspectable& sender, const winrt::TappedRoutedEventArgs& args)
+{
+    if (auto nvi = sender.try_as<winrt::NavigationViewItem>())
+    {
+        if (IsSettingsItem(nvi))
+        {
+            OnSettingsInvoked();
+        }
+        else
+        {
+            OnNavigationViewItemInvoked(nvi);
+        }
+        nvi.Focus(winrt::FocusState::Pointer);
+    }
+}
+
+void NavigationView::OnNavigationViewItemKeyDown(const winrt::IInspectable& sender, const winrt::KeyRoutedEventArgs& args)
+{
+    auto key = args.Key();
+
+    if (auto nvi = sender.try_as<winrt::NavigationViewItem>())
+    {
+        if (IsSettingsItem(nvi))
+        {
+            // Because ListViewItem eats the events, we only get these keys on KeyDown.
```

Comment:
>KeyDown [](start = 79, length = 7)

you could use preview key up if design wants the to be a key up behavior

Reply:
see my response in https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r364018648